src/test/kotlin/de/nordgedanken/auto_hotkey/ide/documentation/AhkDocumentationProviderTest.kt

```diff
@@ -0,0 +1,103 @@
+package de.nordgedanken.auto_hotkey.ide.documentation
+
+import com.intellij.openapi.projectRoots.impl.ProjectJdkImpl
+import com.intellij.psi.PsiElement
+import de.nordgedanken.auto_hotkey.AhkBasePlatformTestCase
+import de.nordgedanken.auto_hotkey.ProjectDescriptor
+import de.nordgedanken.auto_hotkey.WithOneAhkSdkAsProjDefault
+import de.nordgedanken.auto_hotkey.lang.core.AhkFileType
+import de.nordgedanken.auto_hotkey.project.settings.defaultAhkSdk
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.kotest.matchers.string.shouldContain
+import org.junit.jupiter.api.Test
+import java.io.File
+
+
+class AhkDocumentationProviderTest : AhkBasePlatformTestCase() {
+
+    override fun getTestDataPath(): String = File("src/test/resources/").absolutePath
+
+    private fun configureHomePath() {
+        (project.defaultAhkSdk as ProjectJdkImpl).homePath =
+            "$testDataPath/de/nordgedanken/auto_hotkey/documentation/"
+    }
+
+    fun `test getCustomDocumentationElement`() {
+        myFixture.configureByText(AhkFileType, "WinSet")
+        val element = myFixture.findElementByText("WinSet", PsiElement::class.java)
+        val customDocumentationElement = AhkDocumentationProvider().getCustomDocumentationElement(
+            myFixture.editor,
+            myFixture.file,
+            element,
+            0
+        )
+        customDocumentationElement shouldNotBe null
+        customDocumentationElement shouldBe element
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test getUrlFor for command`() {
```

Comment:
It looks like the following code block is duplicated 5 times within this class for 5 tests:
```
configureHomePath()
myFixture.configureByText(AhkFileType, "A_LineNumber")
val element = myFixture.findElementByText("A_LineNumber", PsiElement::class.java).firstChild
```

This should be extracted into a private setup method that can be called at the beginning of each test.
Eg 
```kotlin
private fun getFirstPsiElementOfFileWithText(fileText: String) { ... }

//within the test
fun `test getUrlFor for command`() {
   val element = getFirstPsiElementOfFileWithText("WinSet")
   val url = AhkDocumentationProvider().getUrlFor(element, element)
   val shouldBe ....
}
```